docs2/_code/input_output_penguins.qmd

```diff
@@ -164,10 +164,18 @@ You can convert Ibis tables to torch tensors.
 pip install torch
 ```
 
-```{python}
```

Comment:
I'm not certain but think `{.python}` is the Quarto convention, though what you changed it to works. this is something a formatter should handle ideally IMO

Reply:
Not a huge fan of the `.` convention myself :)